src/tokens/mainnet.json

```diff
@@ -1924,5 +1924,13 @@
     "symbol": "HFT",
     "decimals": 18,
     "logoURI": "https://assets.coingecko.com/coins/images/26136/large/hashflow-icon-cmc.png"
+  },
+  {
+    "chainId": 1,
+    "name": "Arbitrum",
+    "address": "0xB50721BCf8d664c30412Cfbc6cf7a15145234ad1",
+    "symbol": "ARB",
+    "decimals": 18,
+    "logoURI": "https://arbitrum.foundation/logo.png"
   }
```

Comment:
I am surprised this is `chainId` 1. Is that correct?

should probably include the `bridgeInfo` extension here: https://github.com/search?q=repo%3AUniswap%2Fdefault-token-list%20bridgeinfo&type=code

Reply:
the bridge-utils package we use (when we call `chainify` in buildList.ts)will automatically pull in the mapped token on arbitrum at build time. tested locally to confirm that (same situation happened with the extended list [update](https://github.com/Uniswap/extended-token-list/pull/18/files) a few days ago). This particular token added here is the mainnet ARB from their official [list](https://tokenlist.arbitrum.io/ArbTokenLists/arbitrum_token_token_list.json).

Reply:
the example you linked is when we wanna manually override/set the bridgeInfo instead of relying on the bridge-utils (in cases where bridge-utils is unable to fetch a mapping, for example)